lambeq/rewrite/base.py

```diff
@@ -311,9 +310,9 @@ def rewrite(self, box: Box) -> Diagram:
         dom = left.l @ box.dom @ right.r
         new_box = Box(box.name, dom, cod[i:j+1])
         if left:
-            new_box = Diagram.curry(new_box, n_wires=len(left), left=True)
+            new_box = Diagram.curry(new_box, n=len(left), left=False)
```

Comment:
This isn't an error, but it's fairly confusing that `if left`, `left=False`. What's changed between the two DisCoPy versions that the left value has switched?

Reply:
good catch. @toumix, any ideas?

Reply:
It's one of the changes of DisCoPy v1.0, see here: https://github.com/discopy/discopy/releases/tag/0.1.0
> swap the semantics curry(left=True) and curry(left=False) for more consistency (i.e. rigid.Diagram.curry uses left adjoint when left=True)